### PR TITLE
registers: pmpaddrx: add various getters

### PR DESCRIPTION
**Pull request by beezow**
Add corresponding getters for all the setter functions.

### CODE REVIEW

**Review by FawazTirmizi — APPROVED** [review]

File: src/register/pmpcfgx.rs [hunk 1]
```diff
@@ -66,30 +66,10 @@ impl PmpCfg {
...
     #[cfg(riscv32)]
-    cfgs: [PmpCfg; 4],
+    pub cfgs: [PmpCfg; 4],
 
     #[cfg(riscv64)]
-    cfgs: [PmpCfg; 8],
-}
-
-impl PmpCfgCsr {
-    pub fn get_cfg(&self, index: usize) -> PmpCfg {
```
Comment (furquan-rivos) [inline src/register/pmpcfgx.rs:76]: Were there no users of this getter function?

Comment (beezow) [reply src/register/pmpcfgx.rs:76]: Not that I can find.

**Comment by beezow** [comment]
Forget about this PR, but will need for some hubris updates I have in the works.  @furquan-rivos did you have any other concerns?

File: src/register/pmpcfgx.rs (hunk 1 above)
Comment (furquan-rivos) [reply src/register/pmpcfgx.rs:76]: :+1:

**Comment by furquan-rivos** [comment]
> Forget about this PR, but will need for some hubris updates I have in the works. @furquan-rivos did you have any other concerns?

No other concerns. Thanks!

**Review by furquan-rivos — APPROVED** [review]

**Pull request merged**